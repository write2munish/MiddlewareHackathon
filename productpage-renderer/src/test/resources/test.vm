<html><body><p>${debug}</p><p>
<ul>
#foreach($feature in ${util.features})
<li>${feature}</li>
#end
</ul>
</p></body>/html>