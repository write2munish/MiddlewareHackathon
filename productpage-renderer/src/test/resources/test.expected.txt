<html><body><p>DeBuG!</p><p>
<ul>
<li>dStream: dStream enables you to get information, consistently in the same time by delivering premium image quality with digital clarity and speed.*</li>
<li>Advanced MR: From routine studies to emerging applications Ingenia's explorative tools and advanced diagnostic solutions increase the imaging capabilities of MR by addressing the major healthcare trends in neurology, oncology and cardiology.</li>
</ul>
</p></body>/html>